ui/app/AppLayouts/ActivityCenter/views/ActivityNotificationCommunityBanUnban.qml

```diff
@@ -0,0 +1,67 @@
+import QtQuick
```

Comment:
this file and ui/app/AppLayouts/ActivityCenter/views/ActivityNotificationCommunityKicked.qml seem very similar visually, can we optimize to avoid code duplication?

Reply:
That's just a first step of refactoring. I just kept the notification components as they were, and just fixing some small visual issues. The plan is, when we have the complete redesign ready (milestone 2), start from scratch with the definition of an `ActivityNotificationBase` component and we will get rid of a lot of repeated code and duplications.